_inc/client/traffic/verification-services/google.jsx

```diff
@@ -0,0 +1,218 @@
+								duration: 5000
+							}
+						);
+					}
+				} );
+			}
+		} );
+	}
+
+	handleClickAutoVerify = () => {
+		if ( this.props.fetchingSiteData || this.props.fetchingGoogleSiteVerify ) {
+			return;
+		}
+
+		analytics.tracks.recordEvent( 'jetpack_site_verification_google_auto_verify_click' );
+
+		if ( ! this.props.isConnectedToGoogle ) {
+			requestExternalAccess( this.props.googleSiteVerificationConnectUrl, () => {
+				this.checkAndVerifySite();
+			} );
+			return;
+		}
+
+		this.checkAndVerifySite();
+	};
+
+	handleClickSetManually = event => {
+		analytics.tracks.recordEvent( 'jetpack_site_verification_google_manual_verify_click' );
+
+		this.toggleVerifyMethod( event );
+	};
+
+	toggleVerifyMethod = () => {
+		this.setState( {
+			inputVisible: ! this.state.inputVisible,
+		} );
+	};
+
+	render() {
+		if ( this.state.inputVisible ) {
+			return (
+				<FormLabel
+					className="jp-form-input-with-prefix"
+					key="verification_service_google">
+					<span>{ __( 'Google' ) }</span>
+					<TextInput
+						name="google"
+						value={ this.props.value }
+						placeholder={ this.props.placeholder }
+						className="code"
+						disabled={ this.props.isUpdating( 'google' ) }
+						onChange={ this.props.onOptionChange } />
+					<Button
+						primary
+						type="button"
+						className="jp-form-site-verification-edit-button"
+						onClick={ this.props.onSubmit }>
+						{ __( 'Save' ) }
+					</Button>
+				</FormLabel>
+			);
+		}
+
+		if ( this.props.isSiteVerifiedWithGoogle ) {
+			return (
+				<div>
+					<div
+						className="jp-form-input-with-prefix"
+						key="verification_service_google">
+						<span>{ __( 'Google' ) }</span>
+						<div className="jp-form-site-verification-verified">
+							<Gridicon icon="checkmark-circle" size={ 20 } />
+							{ ' ' }
+							<span>{ __( 'Your site is verified with Google' ) }</span>
+						</div>
+						<Button
+							type="button"
+							className="jp-form-site-verification-edit-button"
+							onClick={ this.toggleVerifyMethod }>
+							{ __( 'Edit' ) }
+						</Button>
+					</div>
+					<div className="jp-form-input-with-prefix-bottom-message" >
+						{
+							__( "Monitor your site's traffic and performance from the {{a}}Google Search Console{{/a}}", {
+								components: {
+									a: <ExternalLink
+										icon={ true }
+										iconSize={ 16 }
+										target="_blank" rel="noopener noreferrer"
+										href={ 'https://search.google.com/search-console?resource_id=https://' + this.props.rawUrl }
+									/>
+								}
+							} )
+						}
+					</div>
+				</div>
+			);
+		}
+
+		const disabled = this.props.fetchingSiteData || this.props.fetchingGoogleSiteVerify || this.props.isVerifyingGoogleSite;
+
+		return (
+			<div
+				className="jp-form-input-with-prefix jp-form-google-label-unverified"
+				key="verification_service_google">
+				<span>{ __( 'Google' ) }</span>
+				<Button
+					primary
+					type="button"
```

Comment:
Super minor, but you don't have to specify `type="button"` since this is the default value.